titiler/xarray/factory.py

```diff
@@ -34,8 +33,8 @@ def xarray_open_dataset(
         "decode_times": decode_times,
         "consolidated": True,
     }
-    if multiscale:
-        xr_open_args["group"] = z
+    if group:
+        xr_open_args["group"] = group
```

Comment:
we don't need to have `group` and `multiscale` options, if a `group` is set we will use it